yt_dlp/extractor/viu.py

```diff
@@ -271,6 +289,68 @@ def _real_extract(self, url):
         url, idata = unsmuggle_url(url, {})
         country_code, lang_code, video_id = self._match_valid_url(url).groups()
 
+        if country_code in ('id'):
```

Comment:
Note to self: This region may not only in `ID` region

Reply:
Have you used this in other regions? It appears that Viu is soon to transition to the ViuOTT system across all regions, which means that any downloads made with the old extractor may no longer be possible. For the time being, their old backend remains operational.

Reply:
`('id')` is not a tuple

Reply:
Yeah, i forgot the `comma` (which i think makes it a tuple, no?. Anyway, even though it is a tuple, i think there's better solution rather than adding the countries manually